Admin/GalleryAdmin.php

```diff
@@ -124,8 +124,13 @@ protected function configureFormFields(FormMapper $formMapper)
                 ))
                 ->add('enabled', null, array('required' => false))
                 ->add('name')
+                ->ifTrue($format)
```

Comment:
I think this should be $formats. Also the indentation could be nested to be clearer.